### PR TITLE
Add HW13

### PR DESCRIPTION
**Pull request by Gritskov-Vitaliy**

### CODE REVIEW

File: HW13/script.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+if [[ $C == "exit" ]] || [[ $C == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+if [ -z $A ] || [ -z $B ] || [ -z $C ]; then
+    tput setaf 2; echo "The value A, B or C is empty"; tput setaf 7
+fi
+
+if [[ $A == "Zero" ]] || [[ $A == "zero" ]]; then
```
Comment (mavrik90) [inline HW13/script.sh]: Too many code for letters. use `declare` instead

Comment (Gritskov-Vitaliy) [reply HW13/script.sh]: Done!
But I didn't find a way to add a character check when using `declare`

File: HW13/script.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+fi
+
+tput setaf 4; echo "Enter the number C"; tput setaf 7 
+read C
+if [[ $C == "exit" ]] || [[ $C == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+declare -i zero=0
```
Comment (mavrik90) [inline HW13/script.sh:3]: Convert input string number to lowercase, than use `declare`

Comment (Gritskov-Vitaliy) [reply HW13/script.sh:3]: Done!

**Comment by Gritskov-Vitaliy** [comment]
Added checking for characters other than e, exit and zero..five, and error output
image

File: HW13/script.sh [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            elif (( $A < $B )); then
+                tput setaf 2; echo "B > A > C"; tput setaf 7
+            fi
+        elif (( $B < $C )); then
+            tput setaf 2; echo "A < B < C"; tput setaf 7
+        fi
+    fi
+}
+
+function check_C {
```
Comment (mavrik90) [inline HW13/script.sh]: Use one function instead of 3. Just pass parameter as input to the function.

File: HW13/script.sh [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    if [[ $C == "exit" ]] || [[ $C == "e" ]]; then
+        tput setaf 1; echo "Exit"; tput setaf 7 
+        exit
+    else
+        C=$(echo $C | awk '{print tolower($0)}')
+        check_C
+    fi
+}
+
+function enter_B {
```
Comment (mavrik90) [inline HW13/script.sh]: Also you can use one function for getting all 3 values. As option you can create create array with 3 elements and go  throw all of those elements

Comment (Gritskov-Vitaliy) [reply HW13/script.sh]: Removed about 30 lines of code

Comment (Gritskov-Vitaliy) [reply HW13/script.sh]: image

**Review by mavrik90 — APPROVED** [review]
LGTM

**Pull request merged**